src/core/ext/xds/xds_listener.cc

```diff
@@ -776,9 +778,13 @@ grpc_error_handle AddFilterChainDataForSourceIpRange(
   } else {
     for (const auto& prefix_range :
          filter_chain.filter_chain_match.source_prefix_ranges) {
+      auto addr_str = grpc_sockaddr_to_string(&prefix_range.address, false);
+      if (!addr_str.ok()) {
+        return GRPC_ERROR_CREATE_FROM_CPP_STRING(
+            std::string(addr_str.status().ToString()));
```

Comment:
No need to construct a new `std::string` here.

Reply:
Done.